.buildkite/scripts/steps/functional/performance_playwright.sh

```diff
@@ -2,20 +2,22 @@
 
 set -uo pipefail
 
-if [ -z "${ITERATION_COUNT_ENV+x}" ]; then
-  ITERATION_COUNT="$(buildkite-agent meta-data get performance-test-iteration-count)"
+if [ -z "${PERF_TEST_COUNT+x}" ]; then
+  TEST_COUNT="$(buildkite-agent meta-data get performance-test-iteration-count)"
 else
-  ITERATION_COUNT=$ITERATION_COUNT_ENV
+  TEST_COUNT=$PERF_TEST_COUNT
 fi
 
-tput setab 2; tput setaf 0; echo "Performance test will be run at ${BUILDKITE_BRANCH} ${ITERATION_COUNT} times"
+tput setab 2; tput setaf 0; echo "Performance test will be run at ${BUILDKITE_BRANCH} ${TEST_COUNT} times"
 
 cat << EOF | buildkite-agent pipeline upload
 steps:
```

Comment:
Can we move the steps from this file directly into `.buildkite/pipelines/performance/nightly.yml`? Or do they need to be separate for some reason?

Reply:
I believe this is the only way to create sub-steps since we are dynamically accepting the TEST_COUNT from user. When we move the steps to nightly.yml we lose the ability to provide run tests n times and we can only hardcode the parallelism in that option.

Reply:
Ok, looks like it can't be a bash file like I thought.